### ISSUE TITLE
Filters works only in the first page

### ISSUE THREAD

**Issue by shinneider** [shinneider/django_admin_search#11]
Diff to easy simulate erro
```
--- a/example/app/example/admin.py
+++ b/example/app/example/admin.py
@@ -11,6 +11,7 @@ from .models import Area
 class AreaAdmin(AdvacedSearchAdmin):
     form = AreaForm
     search_form = AreaSearchForm
+    list_per_page = 1
```

Create a 2 or more entries, after this change for another page (other than the first), and try to user advanced filter modal, nothing happens. not filters applied no errors in browser console or python.

**Comment by shinneider** [comment]
Resolved in the [PR-16](https://github.com/shinneider/django_admin_search/pull/16)

**Comment by shinneider** [comment]
is already in develop and will be send to pip in next release

**Comment by shinneider** [comment]
already uploaded to pip, its available in version 3.1.
